llvm-gofrontend/go-llvm-typemanager.cpp

```diff
@@ -1199,6 +1199,27 @@ Btype *TypeManager::circularTypeAddrConversion(Btype *typ) {
   return nullptr;
 }
 
+llvm::Type *TypeManager::landingPadExceptionType()
+{
+  llvm::SmallVector<llvm::Type *, 2> elems(2);
+  elems[0] = llvmPtrType();
+  elems[1] = llvmInt32Type();
+  return llvm::StructType::get(context_, elems);
+}
+
+llvm::FunctionType *TypeManager::personalityFunctionType()
+{
+  const bool isVarargs = false;
+  llvm::SmallVector<llvm::Type *, 4> elems(4);
```

Comment:
That's interesting, the GCC personality function has five arguments.  I wonder why they are different?

    _Unwind_Reason_Code
    PERSONALITY_FUNCTION (int version,
    		      _Unwind_Action actions,
    		      _Unwind_Exception_Class exception_class,
    		      struct _Unwind_Exception *ue_header,
    		      struct _Unwind_Context *context);